.azure-devops/merge.yml

```diff
@@ -3,180 +3,179 @@
+      vmImage: ${{ parameters.linuxImage }}
+
+    steps:
+      - template: templates/setup-python.yml
+      - template: templates/build-publish-azure-cli-test-sdk.yml
+
+  - job: "run_unit_tests_ubuntu"
+    dependsOn:
+      [
+        "build_and_publish_azure_iot_cli_ext",
+        "build_and_publish_azure_cli_test_sdk",
+      ]
+    pool:
+      vmImage: ${{ parameters.linuxImage }}
+    strategy:
+      matrix:
+        Python39:
+          python.version: "3.9"
+        Python310:
+          python.version: "3.10"
+        Python311:
+          python.version: "3.11"
+        Python312:
+          python.version: "3.12"
+
+    steps:
+      - bash: sudo rm -R -f /usr/local/lib/azureExtensionDir
+
+      - template: templates/run-tests-parallel.yml
+        parameters:
+          pythonVersion: "$(python.version)"
+          runUnitTests: true
+          runIntTests: false
+
+  - job: "run_unit_tests_macOs"
+    dependsOn:
+      [
+        "build_and_publish_azure_iot_cli_ext",
+        "build_and_publish_azure_cli_test_sdk",
+      ]
+    pool:
+      vmImage: "macOS-13"
+
+    steps:
+      - template: templates/run-tests-parallel.yml
+        parameters:
+          pythonVersion: "3.12"
+          runUnitTests: true
+          runIntTests: false
+
+  - job: "run_unit_tests_windows"
+    dependsOn:
+      [
+        "build_and_publish_azure_iot_cli_ext",
+        "build_and_publish_azure_cli_test_sdk",
+      ]
+    pool:
+      vmImage: "windows-2019"
+
+    steps:
+      - task: PowerShell@2
+        inputs:
+          targetType: "inline"
+          script: 'ren "C:\Program Files\Common Files\AzureCliExtensionDirectory" "C:\Program Files\Common Files\AzureCliExtensionDirectory1"'
+
+      - template: templates/run-tests-parallel.yml
+        parameters:
+          pythonVersion: "3.12"
+          runUnitTests: true
+          runIntTests: false
+
+  - job: "run_style_check"
+    dependsOn:
+      [
+        "build_and_publish_azure_iot_cli_ext",
+        "build_and_publish_azure_cli_test_sdk",
+      ]
+    pool:
+      vmImage: ${{ parameters.linuxImage }}
+
+    steps:
+      - template: templates/install-azure-cli-released.yml
+      - template: templates/setup-python.yml
+      - template: templates/download-install-local-azure-iot-cli-extension-with-pip.yml
+
+      - script: "pylint azext_iot/ --rcfile=.pylintrc"
+        displayName: "Evaluate with pylint"
+        workingDirectory: "."
+
+      - script: "flake8 azext_iot/ --statistics --config=setup.cfg"
+        displayName: "Evaluate with flake8"
+        workingDirectory: "."
+
+  # TODO: Evaluate this style or similar alternative for setting up CLI env
+  - job: "run_azdev_linter_on_command_table"
+    dependsOn: ["build_and_publish_azure_iot_cli_ext"]
+    displayName: "Evaluate IoT extension command table"
+    pool:
+      vmImage: ${{ parameters.linuxImage }}
+
+    steps:
+      - template: templates/evaluate-command-table.yml
+
+  # - job: 'validate_ref_docs'
```

Comment:
can we delete this

Reply:
Have we confirmed they're not using this anymore in the cli-extensions pipeline? We can always re-add it, I don't have an issue removing for now.